package.json

```diff
@@ -56,9 +56,11 @@
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
     "react-native-shimmer-placeholder": "^2.0.9",
+    "react-native-svg": "13.9.0",
     "react-native-tab-view": "^3.5.2",
     "react-redux": "^9.1.0",
-    "uuid": "^9.0.1"
+    "uuid": "^9.0.1",
+    "expo-splash-screen": "~0.20.5"
```

Comment:
Did this appear after npx expo install --fix? @tolgaozgun

Reply:
I don't remember using this

Reply:
Yes, @selimcanglr could you also try npx expo-doctor?

Reply:
eas automatically runs expo-doctor before build to check issues